dev/tasks/build/builder.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+var Builder;
+
+/**
+ * A CKEditor 5 release builder.
+ *
+ * @class Builder
+ */
+module.exports = Builder = function( target ) {
+	/**
+	 * The target directory where to create the build.
+	 *
+	 * **Warning**: if existing, this directory will be deleted before processing.
+	 *
+	 * @type {string}
```

Comment:
Having a talk about doc tags we can also discus if we want to stick with JSDuck, see ckeditor/ckeditor5-design#27.